src/mono/mono/utils/mono-time.c

```diff
@@ -307,22 +312,20 @@ mono_clock_get_time_ns (mono_clock_id_t clk_id)
 void
 mono_clock_init (mono_clock_id_t *clk_id)
 {
-	// TODO: need to implement this function for PC
-	g_assert_not_reached ();
+	// TODO: need to implement this function for Windows
```

Comment:
nit - For any TODOs, would suggest to create a tracking GH issue and add link as part of comment.

Reply:
In a bunch of these cases, it's more meant in the sense of 'you should note this isn't alive on all platforms if you use the function' rather than work that actually needs to happen as the code stands. Not sure it's really worth an issue in that sort of situation, but I don't mind doing it in the future if you feel differently.